core/src/main/java/com/github/dozermapper/core/converters/PrimitiveOrWrapperConverter.java

```diff
@@ -112,12 +112,13 @@ private Converter getPrimitiveOrWrapperConverter(Class destClass, DateFormatCont
 
     public boolean accepts(Class<?> aClass) {
         return aClass.isPrimitive()
-               || Number.class.isAssignableFrom(aClass)
-               || String.class.equals(aClass)
-               || Character.class.equals(aClass)
-               || Boolean.class.equals(aClass)
-               || java.util.Date.class.isAssignableFrom(aClass)
-               || java.util.Calendar.class.isAssignableFrom(aClass);
+                || aClass.isEnum()
+                || Number.class.isAssignableFrom(aClass)
+                || String.class.equals(aClass)
+                || Character.class.equals(aClass)
+                || Boolean.class.equals(aClass)
+                || java.util.Date.class.isAssignableFrom(aClass)
+                || java.util.Calendar.class.isAssignableFrom(aClass);
```

Comment:
Can you add _aClass.isEnum()_ at the end so the diff, is just 1 line, instead of 7

Reply:
Accepted